### PR TITLE
npm script to automate tunnelling on dev environment using Ngrok

### PR DESCRIPTION
**Pull request by benmort**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
     "dev-start": "nodemon -e js,jsx,md,mustache -w ./src --exec ./dev-tools/babel-run -- ./src/server",
     "generate-contacts": "./dev-tools/babel-run ./dev-tools/generate-contacts.js",
     "jest-test": "jest __test__/backend.test.js",
-    "dev": "nf start -w --procfile ./dev-tools/Procfile.dev"
+    "dev": "npm run tunnel | nf start -w --procfile ./dev-tools/Procfile.dev",
+    "tunnel": "node ./tunnel.js"
```
Comment (schuyler1d) [inline package.json:38]: I think we want something like `if [ "$NGROK_AUTH_TOKEN" != \"\" ] ; then node ./tunnel.js; fi` for the tunnel job -- basically it should be able to work without ngrok.

File: .env.example [hunk 2]
```diff
@@ -46,3 +46,6 @@ EMAIL_HOST_PASSWORD=
 EMAIL_HOST_USER=
 EMAIL_HOST_PORT=
 EMAIL_FROM=
+PORT=_integer_
```
Comment (schuyler1d) [inline .env.example:49]: Does the `_integer_` etc provide info to some IDE?  If so, this is cool.  However, if it's just a textual hint, I'd rather they be blank in this file, so when people copy it over, if they don't modify the line at all, then it stays an empty value (and e.g. tests if the value exist will do the right thing)

**Review by schuyler1d — CHANGES_REQUESTED** [review]
changes just to make sure it's truly optional

**Comment by schuyler1d** [comment]
reopening after moving to main

**Comment by Frydafly** [comment]
Closing since this is pretty old, and created an issue https://github.com/MoveOnOrg/Spoke/issues/1951 in order to not abandon this project!

**Pull request closed without merging**